VAMobile/package.json

```diff
@@ -91,7 +91,7 @@
     "@babel/preset-env": "^7.24.7",
     "@babel/preset-typescript": "^7.24.7",
     "@babel/runtime": "^7.24.7",
-    "@department-of-veterans-affairs/eslint-config-mobile": "0.15.0",
+    "@department-of-veterans-affairs/eslint-config-mobile": "0.18.0",
```

Comment:
Did you want to bump this to be issuing linting warnings for Loading indicator? Not sure if that had been prioritized yet.

Reply:
Good call, I moved this back to 0.15.0